api/ImageBitmap.json

```diff
@@ -30,12 +30,10 @@
             "version_added": "37"
           },
           "safari": {
-            "version_added": false,
-            "notes": "See <a href='https://webkit.org/b/182424'>bug 182424</a>."
+            "version_added": "15"
           },
           "safari_ios": {
-            "version_added": false,
-            "notes": "See <a href='https://webkit.org/b/182424'>bug 182424</a>."
+            "version_added": "15"
```

Comment:
![image](https://user-images.githubusercontent.com/32498324/134072154-3699e42d-3325-4afe-a5bb-f8a69cda1cef.png)

Reply:
Moved to https://github.com/mdn/browser-compat-data/pull/14055